test/common/util.js

```diff
@@ -6,10 +6,23 @@ const exquisite = require('exquisite-sst');
 
 let testsDir = '';
 const testFile = 'scenario.js';
+const sources = loadGeoJSONSources();
 
 function loadFiles(directory) {
     testsDir = directory;
-    return glob.sync(path.join(directory, '**', testFile));
+    let files = [];
+    let fFiles = [];
+    const allFiles = glob.sync(path.join(directory, '**', testFile));
+    allFiles.forEach(function (file) {
+        const name = getName(file);
+        if (name.indexOf('/f-') !== -1) {
```

Comment:
[startsWith](https://developer.mozilla.org/es/docs/Web/JavaScript/Referencia/Objetos_globales/String/startsWith)

Reply:
or (includes)[https://developer.mozilla.org/es/docs/Web/JavaScript/Referencia/Objetos_globales/String/includes]

Reply:
startsWith does not work. includes is a better option ;)